examples/src/main/scala/sttp/tapir/examples/optional_content/optionalValueExample.scala

```diff
@@ -0,0 +1,131 @@
+// {cat=Optional value; effects=Future; server=Pekko HTTP; JSON=circe; docs=Swagger UI}: Optional returned from the server logic, resulting in 404 if None
```

Comment:
Let's put it in the "Error handling" category  (https://tapir.softwaremill.com/en/latest/examples.html)

Reply:
done